### PR TITLE
Upgrade CTST dependencies

### PR DESCRIPTION
**Pull request by KillianG**

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Review by welansari — APPROVED** [review]

File: tests/zenko_tests/Dockerfile [hunk 1]
```diff
@@ -1,11 +1,11 @@
-FROM ubuntu:18.04
+FROM ubuntu:22.04
 
 ENV MOCHA_TAGS not:flaky
 ENV LANG C.UTF-8
 
 # Add Node.js PPA
 RUN apt-get update && apt install -y curl
-RUN curl -sL https://deb.nodesource.com/setup_16.x | bash -
+RUN curl -sL https://deb.nodesource.com/setup_18.x | bash -
```
Comment (francoisferrand) [inline tests/zenko_tests/Dockerfile:8]: can we bump to node 20 ?

**Review by francoisferrand — APPROVED** [review]

File: tests/zenko_tests/Dockerfile (hunk 1 above)
Comment (KillianG) [reply tests/zenko_tests/Dockerfile:8]: Already tried and its not that simple, node-gyp is failing with node 20 actually, i created a ticket for that
https://scality.atlassian.net/browse/ZENKO-4888

**Review by benzekrimaha — APPROVED** [review]

**Pull request merged**